WORKSPACE

```diff
@@ -30,6 +30,36 @@ load("@io_bazel_rules_bison//bison:bison.bzl", "bison_register_toolchains")
 
 bison_register_toolchains()
 
+load("//third_party/gems:rules.bzl", "gemfile_lock_deps")
+
+gemfile_lock_deps(
+    name = "gems",
+    gemfile_locks = [
+        "//gems/sorbet/test/snapshot:{}/src/Gemfile.lock".format(test)
+        for test in [
```

Comment:
this is temporary, right? 
can we leave a TODO?

Reply:
Ultimately it would be good to discover the `Gemfile.lock` files, but I'm not sure how to search the project during the loading phase. I'll add a TODO to this effect.

Reply:
Isn't this going to be a big footgun? What will be the signal that I need to change this list?

I'd love to figure out if we can discover these with some sort of glob.

Reply:
`glob` isn't available during the loading phase, so we'll have to look into ways to do this portably from a script.

Reply:
ack that's too bad. In that case the TODO seems good enough.